tools/rebuild-wheel.py

```diff
@@ -0,0 +1,23 @@
+from wheel.wheelfile import WheelFile
```

Comment:
Add `#!/usr/bin/env python3` shebang and copyright

Reply:
Does this add a requirement on a wheel package? If so, add to tools/requirements.txt

Reply:
Added the wheel package to `requirements-manylinux.txt`